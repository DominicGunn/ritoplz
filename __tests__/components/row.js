'use strict'

/* eslint-env jest */

import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import Row from '../../components/ui/row'

describe('ui row component', () => {
  it('should have children props passed', () => {
    const wrapper = shallow(<Row><h1>Nice</h1></Row>)
    expect(wrapper.find('div').props().children[0]).toEqual(<h1>Nice</h1>)
  })

  describe('snapshots', () => {
    it('should render row', () => {
      const tree = renderer.create(<Row>Nice</Row>).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
