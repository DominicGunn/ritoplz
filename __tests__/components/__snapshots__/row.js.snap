// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui row component snapshots should render row 1`] = `
<div
  className="row"
  data-jsx={2596042597}
>
  Nice
</div>
`;
